sysvabi64/sysvabi64.rst

```diff
@@ -363,6 +365,16 @@ AArch64 system.
 Low Level Information
 =====================
 
+Hardware Requirements
+---------------------
+
+The AArch64 System V ABI requires the presence of SIMD and FP registers.
+
+Procedure call standard requirements
+------------------------------------
+
+The AArch64 System V ABI uses The Base Procedure Call Standard from (AAPCS64_).
```

Comment:
There are some other ABI variations mentioned in the `The standard variants` section of AAPCS64 (fp16 format, sizes of some types), should we mention any of them here too? I think it would be enough to say that the `LP64` data model must be used, maybe with an option to use `ILP32`.

Reply:
I'll have a think. I am a little worried that the statement is a bit too strong, and could rule out things like the vector PCS.

The opening paragraph of https://github.com/ARM-software/abi-aa/blob/main/aapcs64/aapcs64.rst#6the-base-procedure-call-standard has:
```
Application code is expected to conform to one of three data models defined in this standard; ILP32, LP64 or LLP64.
```
I think that is something close to what I want to say, without duplicating the text exactly. Perhaps I can find a way of referencing it more directly.